open_dread_rando/door_patcher.py

```diff
@@ -320,4 +333,59 @@ def update_minimap_for_shield(self, shield: Container, shield_type: ActorData, d
     def remove_all_shields(self):
         for scenario in ["s010_cave", "s020_magma", "s030_baselab", "s040_aqua", "s050_forest", "s060_quarantine", "s070_basesanc", "s080_shipyard", "s090_skybase"]:
             bmmap = self.editor.get_scenario_map(scenario)
-            bmmap.raw.Root.mapBlockages = Container()
\ No newline at end of file
+            bmmap.raw.Root.mapBlockages = Container()
+    
+    def rename_all_shields(self):
+        for scenario in ["s010_cave", "s020_magma", "s030_baselab", "s040_aqua", "s050_forest", "s060_quarantine", "s070_basesanc", "s080_shipyard", "s090_skybase"]:
```

Comment:
Do we really not have a list of all scenarios anywhere? If we don't we absolutely should. Probably on a new file so multiple places can use. (Both `static_fixes.py` and `lua_editor.py` could use this shared list).